LR_1/task_1.py

```diff
@@ -0,0 +1,11 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+i = 0
+kol = 0
+str = 50 * 25 * 4 * 100
+while i<1509949.44:
```

Comment:
Верное решение!
Отличное использование цикла!
Но внесу пару корректировок:
Будет немного лучше, если Вы избавитесь от "магических" чисел, создав несколько переменных под каждое число в задаче.
Например:
```suggestion
disk_mb = 1.44
pages = 100
lines_per_page = 50
chars_per_line = 25
```
И так далее.
Таким образом, ваш код будет более понятен человеку, который не знает суть Вашего задания. Код станет более читаемым.